### ISSUE TITLE
Hydrant symbology: change to reflect status

### ISSUE THREAD

**Issue by rssidlowski** [rssidlowski/Hydrants#8]
Change the symbology for hydrant depending on the status value.

Valid values are:
in service
out of service
in service needs repairs
in service pending unit inspection
unknown

**Comment by rssidlowski** [comment]
The different hydrant symbols should not be just a color change since the user could be color blind.

**Comment by rssidlowski** [comment]
Symbology was adjusted, but have the Hydrants legend at the top and it is not accurate.  Changed the legend to be not only color coded but there are different symbols based on status. See current legend:
![legend](https://cloud.githubusercontent.com/assets/5039052/7044784/2d8d9f62-ddc5-11e4-81fa-43f4c5706424.PNG)

Need to update the Hydrants symbology in the Settings.  Currently displays: 
![hydsettings](https://cloud.githubusercontent.com/assets/5039052/7044825/9506130e-ddc5-11e4-89eb-a9b984245b14.PNG)

Symbology developed:
green hydrant with star - In service
gray hydrant with black circle - Null
yellow hydrant with plus sign -  In service repairs needed
red hydrant with square - Out of service

**Comment by rssidlowski** [comment]
Will you be able to do this one before vacation?  Need further instructions?

**Comment by jday3** [comment]
Do we just need to adjust the legend?

**Comment by rssidlowski** [comment]
Yes in several places:
1) on the button on the map
2) in activity viewer
3) in settings

**Comment by jday3** [comment]
We won't be able to update the legend in the Activity Manager and in the settings of the mobile app without quite a bit of work because they are being pulled directly from the map service by default. I will go ahead and update the static legend that is accessible from the map though.

**Comment by jday3** [comment]
The legend accessible from the map toolbar is now updated

**Comment by rssidlowski** [comment]
See below for valid status values and adjust.
There is no need to keep anything beyond the four below:
In service
out of service
in service repairs needed
in service pending unit inspection
The remaining can be updated as follows:
unknown – These can be update to “in service pending unit inspection”
new hydrant – These can be update to “in service pending unit inspection”
hydrant marked for removal – These can be removed if we can confirm that the hydrant no longer exists. Otherwise they should  be updated to “in service pending unit inspection”

**Comment by gerrykelly** [comment]
I updated all the no-longer-used values to “in service pending unit inspection”.  I also updated null status to “in service pending unit inspection”, since these were effectively "unknown".  The 13 'hydrant marked for removal' hydrants can all be found in WA_HYDRANT.  In production, 7 of these had a feature status of REM and one ABD, so I deleted those records from hydrant_status; the other 5 were ACT so I left them in the table as “in service pending unit inspection”.
I removed the no-longer-used values from the popup legend in the UI and resized that window.

**Comment by gerrykelly** [comment]
There is no reasonable way to update the Hydrants symbology in the Settings (in all of our projects, updating the legends to show multi-value symbols is a significant effort, but with hydrants using compound symbols it would not work).  It might make more sense just to not display a legend for hydrants in the Setting and the user understands that is what the popup is for.

**Comment by rssidlowski** [comment]
Yeah we can remove from settings and just have the toggle since we need to toggle between hydrants and hydrants needing inspections.

**Comment by gerrykelly** [comment]
So, to clarify where things stand and what I should do for followup:
The following fields were added to PRODUCTION WA_HYDRANT (all null values):
LOCATION_TYPE Text(50)
STREET_NUM Text(50)
STREET_NAME Text(255)
CAD_ADDR long text !!!!(was added as Text(255))
CROSS_STREET text(255)
CORNER_POSITION text(50)
LOCATION_DESCRIPTION text(50) !!!! but why limit to 50char?
DISTRICT text(6)
HYDR_ZONE long integer
The production WA_HYDRANT was not migrated to the DEV OPERATIONS (I should probably replace the whole dataset at some point), so these fields do not exist in DEV WA_HYDRANT
The fields, ST_NAME, C_ST_NAME, LOCATION_DESCRIPTION, ADDRESS exist ONLY in the featureclass WA_HYDRANTdev that I created to be used during development.  This is the featureclass that is used for the mapservice here and in City DEV.
So, what to do?  Assuming the fields that were added to PRODUCTION WA_HYDRANT are as desired (although I’d request lengthening the 2 fields I commented on) I think I should:
1)	Copy the PRODUCTION WA_HYDRANTS to dev
2)	Populate those fields as best I can
3)	Deliver a script (to Marian?) that populates the fields in the WATER DB, so these values then are part of OPERATIONS and will then be maintained by normal workflow in WATER
a.	That would need to be an approved request that it would become the responsibility of whoever edits WA_HYDRANTS
4)	Recreate the mapservice
         a.	Joe would need to do this at the City
5)	Revise the UI to use these new fields’ equivalent of what I’d had in WA_HYDRANTdev
        a.	And now do we want to add other new fields to the UI?
Please direct.

**Comment by gerrykelly** [comment]
just noting that you will still see hydrants in gray with a null status when it is a hydrant in WA_HYDRANTS with no HYDRANT_STATUS record.  We can either keep this as-is so that the hydrant will not have status until it's first inspection is recorded, or we can insert dummy records with status of “in service pending unit inspection”, or we can just resymbolize nulls as “in service pending unit inspection” in the mapservice

**Comment by gerrykelly** [comment]
I removed Hydrants from the settings page, but added a toggle as described (should work, but we need some overdue inspection hydrants in the hydrants service ... I'll work with Joe on this later)

**Comment by gerrykelly** [comment]
Also, unlike for all hydrants, I think the overdue ones should display at any scale
